packages/react/src/Heading/index.ts

```diff
@@ -1,2 +1,2 @@
-export { Heading } from './Heading'
+export { Heading, headingLevels, headingSizes } from './Heading'
 export type { HeadingLevel, HeadingProps } from './Heading'
```

Comment:
User can get to `HeadingLevel` with `HeadingProps["level"]`. Do we want to export them separately?

Reply:
I like how it makes a ‘heading level’ a thing of its own. And using `HeadingProps` imports the entire type object instead of one property, although I don’t know how bad that is.

Slightly out of scope, but I’ve removed the separate type now. That does make this a breaking change; if that’s a reason to leave it in we can revert the commit.